test/types.js

```diff
@@ -298,11 +298,11 @@ defineTypeTest('datetime_fraction', [
 ], '5.6.4');
 
 defineTypeTest('timestamp_fractional', [
-  'TIMESTAMP(3) NULL',
+  'TIMESTAMP(3) NULL'
 ], [
   ['"1970-01-01 00:00:01.123"'],
   ['"2038-01-18 03:14:07.900"'],
-  ['"2014-12-27 01:07:08.001"'],
```

Comment:
@nevill I see a lot of people using the comma, some linters complain if it's there and others complain if it's not. I'll revert these eventually.